openedx/core/djangoapps/content_libraries/docs/decisions/0003-course-content-independence.rst

```diff
@@ -0,0 +1,41 @@
+3. Course Content should not depend on Library Content
+------------------------------------------------------
```

Comment:
could you update this heading to `#####` and the others headings to `*****`? https://docs.openedx.org/en/latest/documentors/references/quick_reference_rst.html#headings

Reply:
Sure. I was just copying the other ADRs in that folder. I'll update them as well in a separate commit.